src/App.js

```diff
@@ -1,61 +1,68 @@
 import styled from 'styled-components';
 import { nanoid } from 'nanoid';
 import { Route, Routes } from 'react-router-dom';
-import { useState } from 'react';
 import useLocalStorage from './hooks/useLocalStorage.js';
 import Navigation from './Navigation.js';
 import WishlistPage from './pages/WishlistPage.js';
+import BookmarksPage from './pages/BookmarksPage.js';
 import AddWishPage from './pages/AddWishPage.js';
 
 export default function App() {
   const [diveWishes, setDiveWishes] = useLocalStorage('DivingWishlist', []);
-  const [isDialogVisible, setIsDialogVisible] = useState(false);
-  const [currentWishId, setCurrentWishId] = useState('');
 
   return (
     <AppGrid>
-      <Header>
-        <h1>Diving Wishlist</h1>
-      </Header>
-      <main>
-        <Routes>
-          <Route
-            path="/"
-            element={
-              <WishlistPage
-                diveWishes={diveWishes}
-                confirmDeleteWish={confirmDeleteWish}
-                cancelDeleteWish={cancelDeleteWish}
-                isDialogVisible={isDialogVisible}
-                showDeleteDialog={showDeleteDialog}
-              />
-            }
-          />
-          <Route
-            path="/add-wish"
-            element={<AddWishPage onAddDiveWish={addToWishlist} />}
-          />
-        </Routes>
-      </main>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <WishlistPage
+              diveWishes={diveWishes}
+              onToggleBookmark={toggleBookmark}
+              onDeleteDiveWish={handleDeleteWish}
+            />
+          }
+        />
+        <Route
+          path="/favorites"
+          element={
+            <BookmarksPage
+              bookmarkedWishes={diveWishes.filter(
+                diveWish => diveWish.isBookmarked === true
+              )}
```

Comment:
For readability I prefer to save the filtered elements in a `const` just before the `return` statement and then just pass it. It's basically the same and just my personal preference :D

Reply:
I actually did it that way and then changed it again :D